stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md

```diff
@@ -77,9 +77,9 @@ match ret3 {
 }
 ```
 
-### Inserting with Returning generated keys
+### Inserting data with auto-generated keys
 
-Following is an example of inserting data, while returning the auto generated keyes by executing an INSERT statement using the `updateWithGeneratedKeys` operation.
+This example demonstrates inserting data while returning the auto-generated keys by executing an INSERT statement that uses the `updateWithGeneratedKeys` operation.
```

Comment:
Please re-evaluate the usage of commas.

Reply:
No comma, but you might want to break this up into two sentences to make it easier to parse. "This example demonstrates inserting data while returning the auto-generated keys. It achieves this by executing an INSERT statement that uses the `updateWithGeneratedKeys` operation."

Reply:
I think this sentence makes more sense if we replace the word "while" with "and":

This example demonstrates inserting data **and** returning the auto-generated keys by executing an INSERT statement that uses the `updateWithGeneratedKeys` operation. Is this okay?

Reply:
Actually, I'm wondering if the meaning is correct. Does the INSERT statement use the operation, or does the operation execute the INSERT statement? If the latter, I'd rewrite this as "This example demonstrates inserting data while returning the auto-generated keys. It achieves this by using the `updateWithGeneratedKeys` operation to execute the INSERT statement."

Reply:
@dulanjal could you please clarify which meaning we want to give in this sentence?

Reply:
Checked with @dulanjal. Yes, the operation executes the INSERT statement. Changed this sentence as suggested by @appellj with this commit: https://github.com/ballerina-platform/ballerina-lang/pull/8071/commits/b100133f6879d70445f59461cd7a1c84ecc1c05b